cwms-data-api/src/main/java/cwms/cda/ApiServlet.java

```diff
@@ -320,6 +325,7 @@ public void init() {
         PolicyFactory sanitizer = new HtmlPolicyBuilder().disallowElements("<script>").toFactory();
         APP_CONTEXT = this.getServletContext().getContextPath();
         javalin = Javalin.createStandalone(config -> {
+                    config.enableDevLogging();
```

Comment:
Did you mean to include Dev Logging?

Reply:
I most definitely did not! Thank you for catching